test/docfx.Test/template/MustacheXrefTagParserTest.cs

```diff
@@ -50,12 +51,60 @@ public class MustacheXrefTagParserTest
             )]
         [InlineData("<xref uid='{{ . }}'/>",
             "{{#href}}" +
-            "  <a href=\"{{href}}\"> {{name}} </a>" +
+            "  <a href='{{href}}'> {{name}} </a>" +
             "{{/href}}" +
             "{{^href}}" +
             "  <span> {{name}} </span>" +
             "{{/href}}"
             )]
+        [InlineData("<xref uid='{{ . }}' title='{{title}}'/>",
+            "{{#href}}" +
+            "  <a href='{{href}}' title='{{title}}'> {{name}} </a>" +
+            "{{/href}}" +
+            "{{^href}}" +
+            "  <span> {{name}} </span>" +
+            "{{/href}}"
+            )]
+        [InlineData(
```

Comment:
#6081 transformed result reads easier to me, while its process logic maybe not that straight-forward :/